charts/renovate/templates/cronjob.yaml

```
@@ -88,7 +88,16 @@ spec:
             - name: {{ .Chart.Name }}
               image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
               imagePullPolicy: {{ .Values.image.pullPolicy }}
-              {{- if or .Values.cronjob.preCommand .Values.cronjob.postCommand}}
+              {{- if or .Values.cronjob.commandOverride .Values.cronjob.argsOverride }}
+              {{- if .Values.cronjob.commandOverride }}
+              command:
+                {{- toYaml .Values.cronjob.commandOverride | nindent 16 }}
+              {{- end }}
+              {{- if .Values.cronjob.argsOverride }}
+              args:
+                {{- toYaml .Values.cronjob.argsOverride | nindent 16 }}
+              {{- end }}
+              {{- else if or .Values.cronjob.preCommand .Values.cronjob.postCommand }}
```

```suggestion
              {{- else }}
```
why the new condition?

---

it's not new, it's was the previous condition.

---

@viceice lil up, just in case you didn't see my responses